news/4.38/pde.md

```diff
@@ -10,7 +10,21 @@ A special thanks to everyone who [contributed to PDE](acknowledgements.md#plug-i
 ## API Tools
 -->
 
-<!--
----
 ## PDE Compiler 
--->
+
+
+### Version Mapping for Required Bundles
```

Comment:
Is it fair to characterize it like this?

Quickfix for Bundle Requirement with Missing Version Constraint

What  you propose really tells me nothing directly meaningful.